tests/distribution_utils/test_distribution_utils.py

```diff
@@ -248,3 +249,44 @@ def test_ensure_positive_ndt():
 
     assert np.all(after_replacement[mask] == np.array(-66.1))
     assert np.all(after_replacement[~mask] == logp[~mask])
+
+
+def test_make_likelihood_callable():
+    """Test the make_likelihood_callable function."""
+
+    def mock_func(*args, **kwargs):
+        return 1, 2, 3
+
+    with patch(
+        "hssm.distribution_utils.dist.make_jax_logp_funcs_from_callable",
+        return_value=("mocked_function", "mocked_grad", "mocked_nojit"),
+    ) as mock_make_jax_logp_funcs:
+        distribution_utils.make_likelihood_callable(
+            loglik=mock_func,
+            loglik_kind="analytical",
+            backend="jax",
+            params_only=False,
+        )
+
+        mock_make_jax_logp_funcs.assert_called_once_with(
+            mock_func, vmap=False, params_only=False
+        )
+
+    with patch(
+        "hssm.distribution_utils.dist.make_jax_logp_funcs_from_callable",
+        return_value=("mocked_function", "mocked_grad", "mocked_nojit"),
+    ) as mock_make_jax_logp_funcs:
+        distribution_utils.make_likelihood_callable(
+            loglik=mock_func,
+            loglik_kind="approx_differentiable",
+            params_is_reg=[False, False, False, False],
+            backend="jax",
+            params_only=False,
+        )
+
+        mock_make_jax_logp_funcs.assert_called_once_with(
+            mock_func,
+            vmap=True,
+            params_only=False,
+            params_is_reg=[False, False, False, False],
+        )
```

Comment:
two comments on this block:

1. slightly confused that no returns are stored for `distribution_utils.make_likelihood_callable()` 
2. do you need to enter the same context twice here?